docs/source/asr/speaker_recognition/results.rst

```diff
@@ -62,4 +62,23 @@ Speaker Recognition Models
    :file: data/speaker_results.csv
    :align: left
    :widths: 30, 30, 40
-   :header-rows: 1
\ No newline at end of file
+   :header-rows: 1
+
+
+   
```

Comment:
Inference should be above models table. IE model tables should be after all other documentation.